### ISSUE TITLE
bug

### ISSUE THREAD

**Issue by gabrieltf-hub** [elias-sundqvist/obsidian-annotator#47]
if comment have two or more ‘\n’, annotation will disappear when markdown file from MD to Annotate.

**Comment by elias-sundqvist** [comment]
Good catch! I'll fix this and try to make some unit tests so that these kinds of issues are less likely to pop up in the future.

**Comment by elias-sundqvist** [comment]
This should be fixed in the new release :)
